ext/date/tests/bug48678.phpt

```diff
@@ -17,15 +17,9 @@ DateInterval Object
     [i] => 30
     [s] => 5
     [f] => 0
-    [weekday] => 0
-    [weekday_behavior] => 0
-    [first_last_day_of] => 0
     [invert] => 0
     [days] => 
-    [special_type] => 0
-    [special_amount] => 0
-    [have_weekday_relative] => 0
-    [have_special_relative] => 0
+    [from_string] => 
 )
```

Comment:
This is more of a nit but would it be possible to use ``var_dump()`` instead of ``print_r()`` as now the test as a trailing whitespace which can get removed by IDEs or even git.

Also I did lead me to having a double take about the value because ``false`` gets converted to an empty string.

But feel free to ignore this one can always change this in a follow up commit.

Reply:
I would have done that for new code, but not for existing tests. FWIW, doesn't the test framework ignore trailing spaces anyway?

Reply:
I don't think it does.